services/ui-src/src/components/pages/ReviewSubmit/McparReviewSubmitPage.tsx

```diff
@@ -283,4 +294,18 @@ const sx = {
     width: "100%",
     justifyContent: "space-between",
   },
+  alert: {
+    marginBottom: "2rem",
+  },
+  submitButton: {
+    "&:disabled": {
+      opacity: 1,
+      background: "#d9d9d9",
+      color: "#5A5A5A",
+
+      "&:hover": {
+        background: "#d2d2d2",
+      },
```

Comment:
Request: Use color vars from theme.ts.

- #d9d9d9 = "palette.gray_lighter"
- #5a5a5a = "palette.gray"
- #d2d2d2 = _does not exist in current color scheme_ -- check with design to see if they want #f2f2f2 (gray_lightest) or something else.

Reply:
Talked with Kim and noted there is no fancy hover state for disabled buttons. Removed the hover and swapped to using palette.